_includes/releases/v22.1/v22.1.0-alpha.2.md

```diff
@@ -0,0 +1,640 @@
+- If graceful drain range lease transfer encounters issues, verbose logging is now automatically enabled to help with troubleshooting. [#68488][#68488]
+- All [`cockroach` commands](../v22.1/cockroach-commands.html) now log their stack but do not exit when sent a `SIGQUIT` signal. This behavior is consistent with the behavior of `cockroach start`. [#75678][#75678]
+- The [`debug zip`](../v22.1/cockroach-debug-zip.html) utility now also scrapes the cluster-wide KV replication reports in the output. [#75239][#75239]
+- The flag `--self` of the [`cockroach node decommission` command](../v22.1/cockroach-node.html) is deprecated. Instead, operators should specify the node ID of the target node as an explicit argument. The node that the command is connected to should not be a target node. [#74319][#74319]
+- Added a new optional `version` argument to the `doctor examine` command. This can be used to enable or disable validation when examining older ZIP directories. [#76166][#76166]
+- The `debug zip` CLI command now supports exporting `system` and `crdb_internal` tables to a ZIP folder for tenants. [#75572][#75572]
+- Added instructions to an error message when initializing `debug tsdump`. [#75880][#75880]
+- `cockroach sql` (and [`demo`](../v22.1/cockroach-demo.html)) now continue to accept user input when Ctrl+C is pressed at the interactive prompt and the current input line is empty. Previously, it would terminate the shell. To terminate the shell, the client-side command `\q` is still supported. The user can also terminate the input altogether via `EOF` (Ctrl+D). The behavior for non-interactive use remains unchanged. [#76427][#76427]
+- The interactive SQL shell (`cockroach sql`, `cockroach demo`) now supports interrupting a currently running query with Ctrl+C, without losing access to the shell. [#76437][#76437]
+- Added a new CLI flag `--max-tsdb-memory` used to set the memory budget for timeseries queries when processing requests from the [**Metrics** page in the DB Console](../v22.1/ui-overview-dashboard.html). Most users should not need to change this setting as the default of 1% of system memory or 64 MiB, whichever is greater, is adequate for most deployments. In  cases where a deployment of hundreds of nodes has low per-node memory available (for example, below 8 GiB) it may be necessary to increase this value to `2%` or higher in order to render timeseries graphs for the cluster using the DB Console. Otherwise, use the default settings. [#74662][#74662]
+- Node drains now ensure that SQL statistics are not lost during the process, but are now preserved in the statement statistics system table. [#76397][#76397]
+- The CLI now auto completes on tab by using `SHOW COMPLETIONS AT OFFSET`. [#72925][#72925]
+
+<h3 id="v22-1-0-alpha-2-api-endpoint-changes">API endpoint changes</h3>
+
+- Created a new endpoint `/sqlroles` that returns a list of the SQL roles for the SQL user logged in. [#74920][#74920]
```

Comment:
This was backported to 21.2.5 but I was told to drop it from the release notes because we don't document the endpoint.  It's for internal use really.